R/musMusculus/mouseAtlas.r

```diff
@@ -52,6 +52,15 @@ appendGenes <- function(mouseGenesVector, GSEMatrix)
   return(fullMatrix) #Return fullMatrix
 }
 
+source("~/Reference-Matrix-Generation/R/utils/check.r")
+
+checkRawCounts(GSE113049, 50)
+checkRawCounts(GSE124952)
+checkRawCounts(GSE143435_D0)
+checkRawCounts(GSE143435_D2)
+checkRawCounts(GSE143435_D5)
+checkRawCounts(GSE143435_D7)
+
 ref_mats <- list(GSE113049, GSE124952, GSE143435_D0, GSE143435_D2, GSE143435_D5, GSE143435_D7)
```

Comment:
I suggest that you generate the list of matrices before calling checkRawCounts. That way you can loop over the list, check for raw counts, then remove matrices that don't have raw counts if necessary.

i.e. (pseudocode)
```r
ref_mats <- list(GSE113049, GSE124952, GSE143435_D0, GSE143435_D2, GSE143435_D5, GSE143435_D7)

# iterate over the list and check the matrices
# might need to use `sapply` instead of `lapply` to get a vector of logicals
is_counts <- lapply(ref_mats, 
  function(x){
    check_raw_counts(x) == "raw_counts"
})
# subset the list using the logical vector
ref_mats <- ref_mats[is_counts]
```

Reply:
See commit 88a27d5